package.json

```diff
@@ -25,6 +25,7 @@
     "lakebtc_nodejs": "0.1.x",
     "lodash": "2.x",
     "moment": "2.4.x",
+    "nodejs-websocket": "1.6.0",
```

Comment:
So the webserver is a Gekko "plugin". Plugins dependencies are optional (you only need them if you actually use them. Gekko has detection for missing dependencies that are required for plugins you want to use. Just put the dependency in [here](https://github.com/askmike/gekko/blob/stable/plugins.js#L121-L124).